skia/BUILD.gn

```diff
@@ -36,6 +36,7 @@ skia_support_gpu = !is_ios
 skia_support_pdf = false
 skia_support_skottie = false
 enable_skia_wuffs_gif = true
+skia_support_skparagraph = true
```

Comment:
skparagraph should be a default requirement for Text Component. Do we need to add this flag?

Reply:
As skparagraph is required to enabled by default, removed this flag